oaknational/Oak-Web-Application
# Feat/lesq 1110/search results styling [LESQ-1110]

## Description

Music year: 2004

- refactor search results item to be a clickable card rather than just the button/link on the card
- update styling for selected, expanded, hover and focus states of cards
- update spacing on search results page
- clean up component no longer in use

## How to test

1. Go to https://deploy-preview-2907--oak-web-application.netlify.thenational.academy/teachers/search?term=jekyll
2. You should see new hover, focus and expanded states matching designs: https://www.figma.com/design/YcWQMMhHPVVmc47cHHEEAl/Oak-Design-Kit?node-id=8890-42311&node-type=canvas&m=dev 


## ACs

- [ ] matches designs
- [ ] events still fire (search result expanded and search result opened)
- [ ] links still work
- [ ] navigable by keyboard
